src/vstest.console/CommandLine/CommandLineOptions.cs

```diff
@@ -282,13 +283,29 @@ internal Framework TargetFrameworkVersion
         /// <param name="source">Path to source file to look for tests in.</param>
         public void AddSource(string source)
         {
+            var extnList = new List<string>() { ".xap", ".appx", ".dll", ".exe" };
```

Comment:
>appx [](start = 58, length = 4)

add "*.appxrecipe" to this list

Reply:
Added it to the list and pushed, as per Sarab's feedback the fix that I'm trying may not be the perfect one So let's meet again tomorrow and identify the fix.
Thank you for your help.